### PR TITLE
BGDIINF_SB-1971: Added KML author in POST request

### PR DESCRIPTION
**Pull request by ltshb**
Now in the POST request you can specify the KML author. This will allow
to differentiates KMLs created by user (map viewer) to the one created
by E2E tests.

### CODE REVIEW

File: app/helpers/dynamodb.py [hunk 1]
```diff
@@ -40,7 +40,7 @@ def __init__(self, table_name, bucket_name, endpoint_url, table_region):
         self.bucket_name = bucket_name
         self.endpoint = endpoint_url
 
-    def save_item(self, kml_id, kml_admin_id, file_key, timestamp, empty=False):
+    def save_item(self, kml_id, kml_admin_id, file_key, timestamp, empty=False, author=''):
```
Comment (boecklic) [inline app/helpers/dynamodb.py:43]: see comment in other PR: don't you wanna use 'mapviewer' or sth as default? Or would you add this in the viewer then?

Comment (ltshb) [reply app/helpers/dynamodb.py:43]: See comment https://github.com/geoadmin/doc-api-specs/pull/15#discussion_r713771088

**Review by hansmannj — APPROVED** [review]
Good idea.
Just need to make sure, this will not cause troubles w.r.t. backwards-compatibility.
Especially when migrating the "old" kmls, that don't have the `author` field yet. Should be possible to simply batch-fill them with "" in that case, I guess.

File: app/helpers/dynamodb.py (hunk 1 above)
Comment (ltshb) [reply app/helpers/dynamodb.py:43]: Changed default to `unknown`

**Comment by ltshb** [comment]
> Good idea.
> Just need to make sure, this will not cause troubles w.r.t. backwards-compatibility.
> Especially when migrating the "old" kmls, that don't have the `author` field yet. Should be possible to simply batch-fill them with "" in that case, I guess.

This `author` field is only available in DB and not used anywhere. It will help in future to do some cleanup especially if we do E2E test on PROD we could safely delete kml files that came from E2E tests. For the migration we can then set the author to `legacy-mapviewer` for example.

**Pull request merged**